plugins/inputs/socket_listener/README.md

```diff
@@ -29,6 +29,11 @@ This is a sample configuration for the plugin.
   ## Only applies to stream sockets (e.g. TCP).
   ## 0 (default) is unlimited.
   # max_connections = 1024
+  
+  ## Read deadline (timeout).
+  ## Only applies to stream sockets (e.g. TCP).
+  ## 0 (default) is unlimited.
+  # read_deadline = 30
```

Comment:
I think this should be `timeout`, the reason is that it is a relative time where a deadline would be absolute and I think the "read" part is implied because of the input type.